app/controllers/sessions_controller.rb

```diff
@@ -19,6 +16,13 @@ def create
 
   private
 
+  def find_user(session)
+    who, pass = session.fetch(:who), session.fetch(:password)
```

Comment:
Question: if either of these raise a `KeyError`, I assume there's some generic 500 response shown in the exceptions_app?